### ISSUE TITLE
ember electron:make does not work

### ISSUE THREAD

**Issue by davideschiera** [felixrieseberg/ember-electron#309]
Possibly duplicate of #308, reposting as it seems to be a blocking issue for using this great tool :-(

To replicate

```
ember new ember-electron-test
cd ember-electron-test
ember install ember-electron
ember electron:make
```

Here the complete output:
```
Davides-MacBook-Pro:ember-electron-test davide$ ember electron:make --platform=darwin

"ember-welcome-page" was detected in your devDependencies!
Please note that this addon only works in development environment
and will not render in production mode. It is safe to uninstall
this addon once you removed the {{welcome-page}} template tag.

'instrument' is imported from external module 'ember-data/-debug' but never used
cleaning up...
Built and assembled Electron project successfully. Stored in "/Users/davide/Documents/ember-electron-test/tmp/package_task-tmp_path-dQSr6NMz.tmp".
Packaging Electron project.
✖ Compiling Application

An unhandled error has occurred inside Forge:
Cannot find module '/Users/davide/Documents/ember-electron-test/tmp/package_task-tmp_path-dQSr6NMz.tmp/node_modules/electron-compile/lib/cli.js'
Error: Cannot find module '/Users/davide/Documents/ember-electron-test/tmp/package_task-tmp_path-dQSr6NMz.tmp/node_modules/electron-compile/lib/cli.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at _callee2$ (/Users/davide/Documents/ember-electron-test/node_modules/electron-forge/dist/util/compile-hook.js:181:36)
    at tryCatch (/Users/davide/Documents/ember-electron-test/node_modules/regenerator-runtime/runtime.js:65:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/davide/Documents/ember-electron-test/node_modules/regenerator-runtime/runtime.js:299:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/davide/Documents/ember-electron-test/node_modules/regenerator-runtime/runtime.js:117:21)
    at GeneratorFunctionPrototype.tryCatcher (/Users/davide/Documents/ember-electron-test/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/Users/davide/Documents/ember-electron-test/node_modules/bluebird/js/release/generators.js:97:49)
    at /Users/davide/Documents/ember-electron-test/node_modules/bluebird/js/release/generators.js:201:15
    at /Users/davide/Documents/ember-electron-test/node_modules/electron-forge/dist/util/ora-handler.js:70:5
    at Promise.F (/Users/davide/Documents/ember-electron-test/node_modules/core-js/library/modules/_export.js:35:28)
    at asyncOra (/Users/davide/Documents/ember-electron-test/node_modules/electron-forge/dist/util/ora-handler.js:69:10)
    at _callee3$ (/Users/davide/Documents/ember-electron-test/node_modules/electron-forge/dist/util/compile-hook.js:46:45)
    at tryCatch (/Users/davide/Documents/ember-electron-test/node_modules/regenerator-runtime/runtime.js:65:40)
✖ Preparing native dependencies

An unhandled error has occurred inside Forge:
ENOENT: no such file or directory, lstat '/private/var/folders/hx/8sjhky3x2f39ms2x95zd8yc40000gn/T/electron-packager/darwin-x64/ember-electron-test-darwin-x64/Electron.app/Contents/Resources/app/node_modules'
Error: ENOENT: no such file or directory, lstat '/private/var/folders/hx/8sjhky3x2f39ms2x95zd8yc40000gn/T/electron-packager/darwin-x64/ember-electron-test-darwin-x64/Electron.app/Contents/Resources/app/node_modules'
    at Error (native)
Making Electron project.
```

I don't have proofs, but I have the feeling that this was working fine up until few days ago. It might be a dependency updated very recently.

**Comment by davideschiera** [comment]
Just verified that [v2.3.1](https://github.com/felixrieseberg/ember-electron/releases/tag/v2.3.1) works fine.

As I mention in #308, I managed to force the command to work fine by replacing https://github.com/felixrieseberg/ember-electron/blob/5c867a77464dcc875f8e35b9b2d30fddd60a2adf/lib/tasks/assemble.js#L70 with `return 'npm install';`. Maybe @stopdropandrew you have any insights about what could be the issue here?

Thanks!

**Comment by bendemboski** [comment]
Fixed by #316
